spec/grape/api_spec.rb

```diff
@@ -2166,11 +2166,11 @@ def static
     it 'groups nested params and prevents overwriting of params with same name in different groups' do
       subject.desc 'method'
       subject.params do
-        group :group1 do
+        group :group1, type: Array do
```

Comment:
I was acting your comment on the issue not to change the default, if I got that correctly, that is :)